src/Block/BannerBlock.php

```diff
@@ -75,6 +75,22 @@ public function getSummary()
         return '';
     }
 
+    /**
+     * Return content summary for summary section of ElementEditor
+     *
+     * @return array
+     */
+    public function getSummaryData()
+    {
+        $summaryData = [];
```

Comment:
Because this extends `FileBlock` you can have `$summaryData = parent::getSummaryData()` and then delete the next 4 lines 🙂